### PR TITLE
apply hotfixes in Docker images

### PR DESCRIPTION
**Pull request by anestos**

### CODE REVIEW

File: update-images.sh [hunk 1]
```diff
@@ -42,6 +42,19 @@ do
 
   tar -xf "${RELEASE_FILENAME}" -C "${VERSION}"
 
+  if jq -e -r '."'$VERSION'"' hotfixes.json > /dev/null; then
```
Comment (travisspencer) [inline update-images.sh:45]: `... > 2>&1 >/dev/null` ?

File: update-images.sh [hunk 2]
```diff
@@ -42,6 +42,19 @@ do
...
+    # Applying hotfix for $VERSION
+    HOTFIX_PATH=$(jq -e -r '."'$VERSION'".hotfix_path' hotfixes.json)
```
Comment (travisspencer) [inline update-images.sh:47]: on line 45, there's no dot between `VERSION'"` and `hotfix.json`. Is that difference correct?

File: update-images.sh [hunk 3]
```diff
@@ -42,6 +42,19 @@ do
...
+
+    curl -f -s -S -H "Authorization: Bearer ${ACCESS_TOKEN}" "${RELEASE_API}/${VERSION}/${HOTFIX_PATH}/file" > "${HOTFIX_PATH}-${VERSION}.tgz"
+
+    for original_file in $(jq -e -r '."'$VERSION'".original_files[]' hotfixes.json); do
+      rm "${VERSION}/idsvr-${VERSION}/${original_file}"
+    done
+
+    tar -xf "${HOTFIX_PATH}-${VERSION}.tgz" --exclude='*.md' -C "${VERSION}/idsvr-${VERSION}"
```
Comment (travisspencer) [inline update-images.sh:55]: you don't need `-z` to decompress as well as untar?

**Review by travisspencer — APPROVED** [review]

File: update-images.sh (hunk 2 above)
Comment (anestos) [reply update-images.sh:47]: Yes, that `.` is for jq it is there in both lines (47 and 45)

**Pull request merged**